WindSpigot-Server/src/main/java/ga/windpvp/windspigot/async/entitytracker/entry/ThreadSafeCannonEntry.java

```diff
@@ -0,0 +1,49 @@
+package ga.windpvp.windspigot.async.entitytracker.entry;
+
+import java.util.List;
+
+import javax.annotation.concurrent.ThreadSafe;
+
+import ga.windpvp.windspigot.async.AsyncUtil;
+import me.suicidalkids.ion.visuals.CannonTrackerEntry;
+import net.minecraft.server.Entity;
+import net.minecraft.server.EntityHuman;
+import net.minecraft.server.EntityPlayer;
+import net.minecraft.server.EntityTracker;
+
+/*
+ * This is a fast cannon entity entry for entity trackers that is thread safe. All public methods accessed by  
+ * multiple threads are synchronized if not already synchronized.
+ */
+@ThreadSafe
+public class ThreadSafeCannonEntry extends CannonTrackerEntry {
```

Comment:
This class is seemingly missing methods that are synchronized compared to `ThreadSafeEntry`.

Reply:
Oh, I didn't notice, thanks for letting me know